app/assets/javascripts/modules/checkbox_toggle.js

```diff
@@ -0,0 +1,18 @@
+(function(Modules) {
+  "use strict";
+
+  Modules.CheckboxToggle = function() {
+
+    var that = this;
+
+    that.start = function(element) {
+      element.on('change', '.js-checkbox-toggle', toggle);
+
+      function toggle(event) {
+        element.find('.js-checkbox-toggle-target').toggle(element.find('.js-checkbox-toggle').is(':checked'));
+      }
+      toggle();
```

Comment:
I'd move the bit inside the .toggle call into a variable above for readability. Something like:
```js
var isCheckboxChecked = element.find('.js-checkbox-toggle').is(':checked');
element.find('.js-checkbox-toggle-target').toggle(isCheckboxChecked);
```